nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1850,7 +1864,43 @@ private String errMsgForInitializer(Set<Element> uninitFields) {
     return message;
   }
 
-  public static enum MessageTypes {
+  /**
+   * based heavily on {@link ASTHelpers#findSuperMethodInType(Symbol.MethodSymbol, Type, Types)},
+   * but works for constructors
+   */
+  @Nullable
+  private static Symbol.MethodSymbol findSuperConstructorInType(
+      Symbol.MethodSymbol methodSymbol, Type superType, Types types) {
+    Preconditions.checkArgument(methodSymbol.isConstructor(), "only accepts constructor methods");
+    Scope scope = superType.tsym.members();
+    for (Symbol sym : scope.getSymbolsByName(methodSymbol.name)) {
+      if (sym != null
+          && !sym.isStatic()
+          && ((sym.flags() & Flags.SYNTHETIC) == 0)
+          && sym.name.contentEquals(methodSymbol.name)
```

Comment:
Is this always `init` or such? If so, maybe should be comparing with a constant? If not, how are they supposed to be the same? Just curious :)

Reply:
Yeah, this should always be "<init>".  I can clean this up, thx